config/manager/manager.yaml

```diff
@@ -82,6 +82,8 @@ spec:
           value: kata-containers
         - name: RELATED_IMAGE_CAA
           value: quay.io/confidential-containers/cloud-api-adaptor
+        - name: RELATED_IMAGE_PEERPODS_WEBHOOK
+          value: "quay.io/confidential-containers/peer-pods-webhook:latest" 
```

Comment:
It looks a bit inconsistent to have one value with quotes when the one just above hasn't.

According to https://symfony.com/doc/current/reference/formats/yaml.html#strings , strings should be quoted in YAML when containing _special characters_ such as `:` or `-`. This change is thus ok but other values should be fixed in a subsequent PR.